editoast/assets/sprites/generate-atlas.sh

```diff
@@ -4,7 +4,9 @@
 # First add all your svg in a subfolder named to the signaling system (eg: `BAL`)
 # Then run this script. You will need docker.
 
-for signaling_system in *; do
+sprites_directory=$(dirname "$0")
```

Comment:
```suggestion
sprites_directory==$(dirname "$(realpath $0)")
```

Reply:
The `==` is a typo, right?
Should be `sprites_directory=$(dirname "$(realpath $0)")`

Reply:
Yes, my bad